### PR TITLE
Switched to new comp analysis image.

### PR DESCRIPTION
**Pull request by ericbenwa**
New image on WuXi FamilyCODE case study (not sinai).

### CODE REVIEW

**Comment by craigmcginley** [comment]
Deploy preview for *loving-austin-5f2eff* ready!

Built with commit 9523871176b3ad66d691ac0f92dbea9a3de25538

https://deploy-preview-237--loving-austin-5f2eff.netlify.com

File: src/case-studies/wuxi-nextcode-familycode.md [hunk 1]
```diff
@@ -46,15 +46,15 @@ Validated with 50+ individuals, from genetic counselors to clinicians to patient
 
 ## Process
 
-WuXi NextCODE is the leader in powering Precision Medicine efforts around the globe, from Iceland, to China, to England, to the United States. While they are the leaders in academic and government genomics research, they have aspirations to expand into a patient-facing service. While their services worked for academics and governments, WuXi NextCODE knew a different approach was needed to break into the $2.2 billion genetic carrier screening market[^1].
+WuXi NextCODE is the leader in powering Precision Medicine efforts around the globe, from Iceland, to China, to England, to the United States. While they are the leaders in academic and government genomics research, they have aspirations to expand into a patient-facing service. While their services worked for academics and governments, WuXi NextCODE knew a different approach was needed to break into the \$2.2 billion genetic carrier screening market[^1].
```
Comment (craigmcginley) [inline src/case-studies/wuxi-nextcode-familycode.md]: "\" by mistake?

Comment (ericbenwa) [reply src/case-studies/wuxi-nextcode-familycode.md]: Ah, yes... mistake!
I blame Left Shark for that.

Comment (ericbenwa) [reply src/case-studies/wuxi-nextcode-familycode.md]: Just edited that mistake (within Github.com)

**Pull request merged**